docs/blueprint-designer-guide/blueprints.md

```diff
@@ -89,6 +93,13 @@ grain_name:
 Note that in auto-generated blueprints, the grain_name.spec.host.name is automatically exposed as a blueprint input for the blueprint designer ease of use. As best practice, it's recommended to remove the host.name from being an input once the blueprint is exposed to the catalog.
```

Comment:
Maybe instead of exposed the catalog use the Torque term "publish"?
where do we explain what publish means?